src/constants/skins-animations.js

```diff
@@ -533,6 +533,12 @@ const skins = [
     texture: "/head/7cdb1ff8c967c831eb685e09e5dea586ed291472395acf7a25e9d5bb41d6c082",
     release: new Date("2022-07-08 18:00:00 GMT+1").getTime(),
   },
+  {
+    id: "PET_SKIN_BEE_RGBEE",
+    name: "RGBee",
+    texture: "/head/99c6da45da5a46614c05d5f8d7b3dfefb025e82ca4cdb4f0db64ced1315df659",
+    release: new Date("2022-08-26 18:00:00 GMT+1").getTime(),
+  },
```

Comment:
This still needs the animation, I'll try to do it next week if I find some time

Reply:
Okay, didn't know it was animated